CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## `6.0.0` Same as it ever was
+In the past few months we've been able to use `choo@5` a bunch, and people seem
+to like it a lot! In this patch all we're doing is taking choo's existing API,
+and polishing the internals. In theory this means breaking changes, but in
+practice it will mostly mean people need to update their dependencies, and
+things will work great.
+
+Choo v6 includes the upgrades to `bel@5` and `nanomorph@5`. This means up to
+20x faster server rendering, and up to 10x improved browser rendering speeds.
+We've also fixed DOM node caching, and introduced sibling node reordering.
+
+This release also includes an overhauled timing API. We're now capturing more
+events, and are leveraging this in tools like `choo-log` to provide consistent
+feedback on performance.
+
+Choo is now also completely prototype based. It not only means faster execution
+times, and more maintainable codebase - but you can also override methods if
+you ever need to.
+
+We've also tweaked the event system a little. All built-in event names are now
+available as constants under `choo.events`. And we've introduced three new
```

Comment:
Oops, good point - it's two brand new ones, and one that was introduced a
couple of weeks ago and folks might not be aware of. Will fix! :D